package.json

```diff
@@ -14,7 +14,8 @@
     "watch": "onchange -i 'src/**/*' 'docs/src/**/*' -- npm run build",
     "dev-server": "live-server --watch=./docs/build,./docs/index.html --no-browser docs",
     "dev": "sh -c 'npm run watch & npm run dev-server & wait'",
-    "prepublish": "npm run clean && npm run build"
+    "prepublish": "npm run clean && npm run build",
+    "lintfix": "prettier --plugin-search-dir=. src/**/**.js src/**/**.html docs/**/**.js docs/**/**.html --write"
```

Comment:
I don't use semicolons and am satisfied with the current code conventions of this repo. I am open to linting, if it accords with the current conventions. I think it's a good idea to submit a PR about linting independently of a PR for a code update, bugfix, new feature, etc.